### DISCUSSION TITLE
How can I join the libresbc slack?

### DISCUSSION THREAD

**Discussion by jj1682000** [hnimminh/libresbc#86] [General]
as title

**Comment by hnimminh** [comment]
hi @jj1682000 , 
here is the link to join 
https://librertc.slack.com/

**Comment by MikeQc99** [comment]
Hi

Sorry to write in a closed issue but I think it was better than creating a new one related to slack since discussion was just a month ago.

I got an error while trying to join the slack channel. I tried with both my professional and personnel account using gmail login or through email.

Through email, instead of receiving the access code asked by slack, I receive a email that indicate
Subject: No account found
and in the message
Is this email address ok ? Seems like an account of librertc is bind to this address.
try another email address

Through google login, I get an error that I don't have an account yet on this work space.

Is there any configuration blocking use to access this space ? Must you accept user or anything ? It's strange, it's the first time I see this with slack.

**Reply by jj1682000** [reply]
Yes, me too!
sorry, I cant help you about that. You can contact Slack's support, I think.

**Reply by hnimminh** [reply]
Hi please send me your email address (find my email on profile), I will send invitation!

**Comment by MikeQc99** [comment]
I will wait for @hnimminh answer first since, form what I read, some channel are private and you must be invite or accepted.

**Reply by hnimminh** [reply]
Hi please send me your email address (find my email on profile), I will send invitation!
